src/client/actions/test/edit/articleActions.test.ts

```diff
@@ -36,44 +38,63 @@ describe("articleActions", () => {
 
     it("Calls #changeArticle", () => {
       changeArticleData("title", "Title")(dispatch, getState)
-      expect(dispatch.mock.calls[0][0].type).toBe("CHANGE_ARTICLE")
+
+      expect(dispatch).toBeCalledWith({
+        type: "CHANGE_ARTICLE",
+        payload: { data: { title: "Title" } },
+      })
     })
 
     it("Can accept a key/value pair as args", () => {
       changeArticleData("title", "Title")(dispatch, getState)
-      expect(dispatch.mock.calls[0][0].payload.data.title).toBe("Title")
+
+      expect(dispatch).toBeCalledWith({
```

Comment:
I'm not sure how this assertion is different from the one above on line 42.

Reply:
ahh you are totally correct -- updating now

Reply:
This is actually a duplicate test, so going to remove